### PR TITLE
Added midpoint, midpoint_x, midpoint_y attributes to line.

### PR DESCRIPTION
**Pull request by itzpr3d4t0r**

### CODE REVIEW

File: src_c/line.c [hunk 1]
```diff
@@ -234,7 +234,8 @@ pg_line_copy(pgLineObject *self, PyObject *_null)
 }
 
 static PyObject *
-pg_line_is_parallel(pgLineObject *self, PyObject *const *args, Py_ssize_t nargs)
+pg_line_is_parallel(pgLineObject *self, PyObject *const *args,
```
Comment (novialriptide) [inline src_c/line.c:237]: Was this changed via a linter? This change should be reverted.

File: test/test_line.py [hunk 2]
```diff
@@ -285,6 +285,92 @@ def test_attrib_b(self):
...
+
+        self.assertEqual(line.midpoint_x, expected_midpoint[0])
+
+        line.midpoint_x = expected_midpoint[0] - 1
+
+        self.assertEqual(line.midpoint_x, expected_midpoint[0] - 1)
+
+        line = Line(0, 0, 1, 0)
+
+        for value in (None, [], "1", (1,), [1, 2, 3]):
```
Comment (novialriptide) [inline test/test_line.py:339]: We should create a standardized variable for checking these integers & floats in the future, this seems to be a common copy-paste scenario.

**Review by novialriptide — CHANGES_REQUESTED** [review]
Just one change

**Comment by itzpr3d4t0r** [comment]
I don't know what i did here i admit nothing was intentional

File: src_c/line.c (hunk 1 above)
Comment (itzpr3d4t0r) [reply src_c/line.c]: Yes this was my IDE automatic clang formatter

File: test/test_line.py (hunk 2 above)
Comment (itzpr3d4t0r) [reply test/test_line.py]: YEah i agree, some tests are a little more specific tho, so we'll most likely need more than one variant

**Pull request closed without merging**